plugins/modules/vmware_dvswitch.py

```diff
@@ -195,6 +195,12 @@
                 type: int
                 description: Identifies the information related to the switch.
                 default: 0
+            switch_ip:
+                type: str
+                description:
+                    - Assign an IP address to see the distributed switch as a single network device in the NetFlow collector.
+                    - This is instead of as multiple devices corresponding to each host.
+                    - In an IPv6 environment, the ESXi hosts ignore the switch IP address.
```

Comment:
```suggestion
            switch_ip:
                type: str
                description:
                    - Assign an IP address to see the distributed switch as a single network device in the NetFlow collector.
                    - This is instead of as multiple devices corresponding to each host.
                    - In an IPv6 environment, the ESXi hosts ignore the switch IP address.
                version_added: '4.3.0'
```

We should document when the new parameter has been added.